### PR TITLE
GH-1774 Add procedure downloadCSV to MegaChart

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-1774

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage @jchambers-ln Please Review

File: src/composite/MegaChart.js [hunk 1]
```diff
@@ -88,6 +88,11 @@
         return this;
     };
 
+    MegaChart.prototype.downloadCSV = function () {
+        Utility.downloadBlob("CSV", this._chart.export("CSV"));
```
Comment (jchambers-ln) [inline src/composite/MegaChart.js]: does downloadBlob handle user response if for some reason the data stream isn't available?

**Comment by jchambers-ln** [comment]
one note otherwise good-- i

File: src/composite/MegaChart.js (hunk 1 above)
Comment (GordonSmith) [inline src/composite/MegaChart.js]: no idea - but because this is a client side download I suspect that cannot happen?

**Pull request merged**